daemon/inertiad/project/data.go

```diff
@@ -50,6 +52,7 @@ func NewDataManager(dbPath string, keyPath string) (*DeploymentDataManager, erro
 	}
 	if err = db.Update(func(tx *bolt.Tx) error {
 		_, err := tx.CreateBucketIfNotExists(envVariableBucket)
+		_, err = tx.CreateBucketIfNotExists(deployedProjectsBucket)
```

Comment:
we should check for the error on each bucket creation (right now the first one isn't being checked)